.github/workflows/ci.yaml

```diff
@@ -154,8 +154,8 @@ jobs:
           toolchain: nightly-2024-04-15
           components: rustfmt, clippy
           target: aarch64-unknown-linux-gnu,x86_64-unknown-linux-gnu
-      - uses: actions/setup-python@v3 # For http mirroring tests with Flask and FastAPI.
-      - run: pip3 install cargo-zigbuild # For http mirroring test with Flask.
+      - uses: actions/setup-python@v5 # For http mirroring tests with Flask and FastAPI.
+      - run: pip3 install --break-system-packages cargo-zigbuild # For http mirroring test with Flask.
```

Comment:
Is maybe switching to uv better in this case like for towncrier instead of setting `--break-system-packages`?

Reply:
problem is it's quite hard to make everything use uv (requires a venv and activating it..)